### PR TITLE
Updated the Changelog file 

### PR DESCRIPTION
**Pull request by Subhashvysyaraju**
## Description

Updated the changelog.md file as per the native SDKs changelogs.

https://github.com/Microsoft/AppCenter-SDK-Android/blob/master/CHANGELOG.md
https://github.com/Microsoft/AppCenter-SDK-Apple/blob/master/CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+* **[Fix]** Fix an issue where concurrent modification of custom properties was not thread safe.
+* **[Fix]** Fix validating and discarding Not a Number (NaN) and infinite double values for custom properties.
+* **[Fix]** Use standard SQL syntax to avoid affecting users with custom SQLite libraries.
+* **[Fix]** Get database page size dynamically to support custom values.
+
+### AppCenterAnalytics
+
+### Android
+
+* **[Feature]** Add API to specify event persistence priority.
```
Comment (MatkovIvan) [inline CHANGELOG.md]: There is no wrapper for it, please remove

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+
+### iOS
+
+* **[Feature]** Add new trackEvent APIs that take priority (normal or critical) of event logs. Events tracked with critical flag will take precedence over all other logs except crash logs (when AppCenterCrashes is enabled), and only be dropped if storage is full and must make room for newer critical events or crashes logs.
```
Comment (MatkovIvan) [inline CHANGELOG.md]: There is no wrapper for it, please remove

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,48 @@
...
+
+    ```objc
+    [[NSUserDefaults standardUserDefaults] registerDefaults:@{ @"NSApplicationCrashOnExceptions" : @YES }];
+    ```
+
+### AppCenterPush
+
+##iOS
+
+* **[Fix]** Fix `push:didReceivePushNotification:` callback not triggered on notification tapped or received in foreground when a `UNUserNotificationCenterDelegate` is set.  If you have implemented this delegate please remove any call to the `MSPush#didReceiveRemoteNotification:` method as it's now handled by the new [User Notification Center Delegate Forwarder](https://docs.microsoft.com/appcenter/sdk/push/ios).
```
Comment (MatkovIvan) [inline CHANGELOG.md]: Should be updated to use C#/Xamarin (function names + link)

Comment (MatkovIvan) [reply CHANGELOG.md]:
```suggestion
* **[Fix]** Fix `PushNotificationReceived` callback not triggered on notification tapped or received in foreground when a `UNUserNotificationCenterDelegate` is set. If you have implemented this delegate please remove any call to the `Push.DidReceiveRemoteNotification` method as it's now handled by the new [User Notification Center Delegate Forwarder](https://docs.microsoft.com/en-us/appcenter/sdk/push/xamarin-ios).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+* **[Fix]** Use standard SQL syntax to avoid affecting users with custom SQLite libraries.
+* **[Fix]** Get database page size dynamically to support custom values.
+
+### AppCenterCrashes
+
+### Android
+
+* **[Fix]** Preventing stack overflow crash while reading a huge throwable file.
+
+### iOS
```
Comment (guperrot) [inline CHANGELOG.md]: Please remove the iOS section for Crashes as we don't support mac os for Xamarin.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,38 @@
...
+
+* **[Fix]** Do not force crash macOS application on uncaught exception. If you need this behavior you can set the special flag yourself:
+
+    ```objc
+    [[NSUserDefaults standardUserDefaults] registerDefaults:@{ @"NSApplicationCrashOnExceptions" : @YES }];
+    ```
+
+### AppCenterPush
+
+##iOS
```
Comment (guperrot) [inline CHANGELOG.md]: Replace `##iOS` by `#### iOS`

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,38 @@
...
+### iOS
+
+* **[Fix]** Fix an issue where concurrent modification of custom properties was not thread safe.
+* **[Fix]** Fix validating and discarding Not a Number (NaN) and infinite double values for custom properties.
+* **[Fix]** Use standard SQL syntax to avoid affecting users with custom SQLite libraries.
+* **[Fix]** Get database page size dynamically to support custom values.
+
+### AppCenterCrashes
+
+### Android
```
Comment (guperrot) [inline CHANGELOG.md]: `### Android` => `#### Android`

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,38 @@
 # App Center SDK for .NET Change Log
 
+## Version 1.11.0
+
+### AppCenter
+
+### iOS
```
Comment (guperrot) [inline CHANGELOG.md]: `### iOS` => `#### iOS`

**Pull request merged**